README.rst

```diff
@@ -22,30 +27,41 @@ For now, this repo is not intergrated with devstack. You'll be running the app l
 
       mkdir -p ~/workspace/
       cd ~/workspace/
-      git clone https://github.com/edx/devstack.git
+      git clone https://github.com/edx/frontend-app-communications.git
 
 2. Install frontend dependencies
 
    .. code-block::
 
       npm i
 
-3. start the devserver. The app will be running at `localhost:8585`, or whatever port you change it too.
+3. start the devserver. The app will be running at ``localhost:1984``, or whatever port you change it too.
 
    .. code-block::
 
-      npm i
+      npm start
```

Comment:
Dumb question -- do we not have to do the `npm i` the first time?

Reply:
you do, that step is listed above this one. for some reason, both the install and starting the dev server had `npm i` as the command

Reply:
you do, that step is listed above this one. for some reason, both the install and starting the dev server had `npm i` as the command

Reply:
Oh, I missed that! Woops. Thanks :)